src/cli/plugin/setting_parser.js

```diff
@@ -1,5 +1,6 @@
 const { resolve } = require('path');
 const expiry = require('expiry-js');
+import intersection from 'lodash';
```

Comment:
Correction, the current line itself wouldn't error, however `intersection` would be the lodash constructor itself rather than the `intersection` function from within lodash.